src/components/NcListItem/NcListItem.vue

```diff
@@ -585,7 +590,12 @@ export default {
 		computedActionsAriaLabel() {
 			return this.actionsAriaLabel || t('Actions for item with name "{name}"', { name: this.name })
 		},
-
+		computedTarget() {
+			if (this.target !== undefined && this.target !== null) {
+				return this.target
+			}
+			return this.href === '#' ? undefined : '_blank'
+		},
```

Comment:
`href` used in the template was not a component instance property but a slot prop. It should not be replaced with `this.href`